dags/openshift_nightlies/tasks/install/openshift.py

```diff
@@ -52,7 +53,6 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease):
             **self.azure_creds,
             **self.openstack_creds,
             **self.rosa_creds,
-            **self.rhacs_creds,
```

Comment:
why delete this line?

Reply:
fixed, I dont know why I removed it, maybe a typo during adding the gcp creds. cood catch!